.github/workflows/pipeline-deploy-app-preprod-prod.yaml

```diff
@@ -0,0 +1,125 @@
+    with:
+      tag: ${{ inputs.tag }}
+
+  quality-check:
+    name: "Quality Check"
+    uses: NHSDigital/uec-dos-management/.github/workflows/quality-checks.yaml@latest
+    needs:
+      [
+        metadata,
+      ]
+    with:
+      environment: ${{ inputs.environment }}
+      workspace: default
+      stacks: '["application", "data_migration", "domain-test"]'
+    secrets: inherit
+
+  deploy-application-infrastructure:
+      needs:
+        [
+          quality-check,
+          metadata,
+        ]
+      uses: NHSDigital/uec-dos-management/.github/workflows/deploy-infrastructure.yaml@latest
+      with:
+        environment: ${{ inputs.environment }}
+        domain: "uec-dos-service-management"
+        workspace: default
+        stacks: '["application", "data_migration", "domain-test"]'
+        action: apply
+        workflow_timeout: 10
+        tag: ${{ inputs.tag }}
+      secrets: inherit
+
+  deploy-application:
+    name: Deploy lambda application code
+    needs:
+      [
+        metadata,
+        deploy-application-infrastructure,
+        derive-artefact-address,
+      ]
+    uses: NHSDigital/uec-dos-management/.github/workflows/deploy-lambda-code.yaml@latest
+    with:
+      directory: 'application'
+      environment: ${{ inputs.environment }}
+      workspace: default
+      artefact_sub_dir: ${{ needs.derive-artefact-address.outputs.artefact_sub_dir }}
+      services:
+        "[
+          'healthcare-services-data-manager',
+          'locations-data-manager',
+          'organisation-affiliations-data-manager',
+          'organisations-data-manager',
+          'questionnaire-responses-data-manager',
+          'questionnaires-data-manager'
+        ]"
+      commit_hash: ${{ needs.derive-artefact-address.outputs.commit_hash }}
+      artefact_bucket_name: ${{ needs.metadata.outputs.artefact_bucket_name }}
+      tag: ${{ inputs.tag }}
+    secrets: inherit
+
+  deploy-data-migration-code:
+    name: Deploy data migration
+    needs:
+      [
+        metadata,
+        deploy-application-infrastructure,
+        derive-artefact-address,
+      ]
+    uses: NHSDigital/uec-dos-management/.github/workflows/deploy-lambda-code.yaml@latest
+    with:
+      directory: 'data_migration'
+      environment: ${{ inputs.environment }}
+      workspace: default
+      artefact_sub_dir: ${{ needs.derive-artefact-address.outputs.artefact_sub_dir }}
+      services:
+        "[
+          'org_data_load',
+          'locations_data_load',
+          'organisation_affiliations_data_load',
+          'service_data_load'
+        ]"
+      commit_hash: ${{ needs.derive-artefact-address.outputs.commit_hash }}
+      artefact_bucket_name: ${{ needs.metadata.outputs.artefact_bucket_name }}
+      tag: ${{ inputs.tag }}
+    secrets: inherit
+
+  slack-notifications:
+    name: Send notification to slack
+    needs: [
+      metadata,
+      quality-check,
```

Comment:
Remove if we don't need quality check to be run

Reply:
Removed